CTCommons/MSTranscription/KeyProvider.cs

```diff
@@ -35,14 +35,17 @@ public KeyProvider(AppSettings appSettings)
             string subscriptionKeys = _appSettings.AZURE_SUBSCRIPTION_KEYS;
             Keys = new List<Key>();
             CurrentVideoIds = new HashSet<string>();
-            foreach (string subscriptionKey in subscriptionKeys.Split(';'))
+            if (subscriptionKeys.Contains(","))
             {
-                Keys.Add(new Key
+                foreach (string subscriptionKey in subscriptionKeys.Split(';'))
                 {
-                    ApiKey = subscriptionKey.Split(',')[0],
-                    Region = subscriptionKey.Split(',')[1],
-                    Load = 0
-                });
+                    Keys.Add(new Key
+                    {
+                        ApiKey = subscriptionKey.Split(',')[0].Trim(),
```

Comment:
It would probably be better to store the split in a variable instead of splitting it twice (just for good practice/style)

Reply:
Yes that bothered me too. I'll fix it.

Reply:
Resolved in https://github.com/classtranscribe/WebAPI/pull/101/commits/3c03811c3bbf81148c0f33e66f3cfafd9fa0f3b4